t/t2315-resource-system.t

```diff
@@ -163,7 +163,7 @@ test_expect_success 'resources can be configured in TOML' '
 	hosts = "foo[3-10]"
 	properties = ["batch"]
 	EOF
-	flux start -s 1 \
+	flux start -s 11 \
```

Comment:
This change will make this test take significantly longer, especially in CI and with coverage enabled. My suggestion would be this instead:
```diff
diff --git a/t/t2315-resource-system.t b/t/t2315-resource-system.t
index b3a0c665f..10ab4c0d1 100755
--- a/t/t2315-resource-system.t
+++ b/t/t2315-resource-system.t
@@ -165,12 +165,12 @@ test_expect_success 'resources can be configured in TOML' '
        EOF
        flux start -s 1 \
                -o,--config-path=$(pwd)/${name},-Slog-filename=${name}/logfile \
-               flux resource list -s up > ${name}/output &&
+               flux resource list -s all > ${name}/output &&
        test_debug "cat ${name}/output" &&
        cat <<-EOF >${name}/expected &&
             STATE PROPERTIES NNODES   NCORES    NGPUS NODELIST
-               up debug           3       12        0 foo[0-2]
-               up batch           8       32        4 foo[3-10]
+              all debug           3       12        0 foo[0-2]
+              all batch           8       32        4 foo[3-10]
        EOF
        test_cmp ${name}/expected ${name}/output
```